src/views/officer-table.j2

```diff
@@ -0,0 +1,29 @@
+<table>
+	<thead>
```

Comment:
Do you mind using 4 spaces for this file instead of tab characters? 😅

Reply:
Whoops! Done in fb8e213